packages/@apphosting/adapter-angular/src/simple-server/server.mjs

```diff
@@ -4,8 +4,9 @@ import { fileURLToPath } from "url";
 const app = express();
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
+const oneYear = 31_557_600_000;
```

Comment:
TIL express maxAge is in milliseconds but let's stick with traditional seconds-calendar year calculation 31_536_000_000

Reply:
Done.